src/mercury_engine_data_structures/formats/property_enum.py

```diff
@@ -6,15 +7,32 @@
 from mercury_engine_data_structures import crc, dread_data
 
 
+class HashSet(enum.Enum):
+    DREAD_PROPERTY = enum.auto()
+    DREAD_FILE_NAME = enum.auto()
+
+    def get_hashes(self) -> tuple[Dict[str, int], Dict[int, str]]:
```

Comment:
`Tuple` if we wanna maintain compatibility with older python (do we?)

Reply:
eh, let's keep it for now